website/src/components/layout/header/header.jsx

```diff
@@ -46,7 +46,7 @@ const Header = ({ ...props }) => {
         <div className="header__content">
           <Link to="/" className="header__logo">
             <NAVLogo />
-            <Systemtittel className="header__title" tag="h1">
+            <Systemtittel className="header__title" tag="h2">
```

Comment:
Denne skjønner jeg ikke helt. Må ikke alle dokumenter ha en h1? Og om man har en h2, så må det være en h1 over den?

Reply:
Stemmer at alle burde ha en <h1>, men denne <h1> burde være i hovedinnholdet. Problemet nå er at vi får to <h1> når man går inn på en komponent. Litt usikker på hva som er "best", ingen <h1> eller to <h1>.

Reply:
Hva med "Designsystemet" som eneste `<h1>` og at artikkeloverskriften er `<h2>`? Hva tenker du om det, @karinaldw?

Reply:
Eventuelt at "Designsystemet" ikke er en overskrift og la artikler begynne på `<h1>`. Tror jeg holder en knapp på det!

Reply:
Hovedoverskriften h1 burde beskrive siden. Det so er litt utfordrende er jo at den i rekkefølgen kommer under "Designsystemet". 

Må tenke litt mer her, men rekker ikke før vi har sendt inn dispensasjonssøknaden.

Reply:
> Eventuelt at "Designsystemet" ikke er en overskrift og la artikler begynne på <h1>. Tror jeg holder en knapp på det!

Hvordan løser vi dette for eks forside @andnorda ? Hvis jeg ikke har misforstått noe da..

Reply:
Hva med å la "Designsystemet" være h1 med mindre man er på en artikkelside? Ser ut som Aftenposten løser det sånn.

Reply:
Virker som en grei løsning det!